rest/taskana-rest-spring/src/main/java/pro/taskana/task/rest/TaskQueryFilterCustomIntFields.java

```diff
@@ -0,0 +1,846 @@
+            Triplet.of(
+                TaskCustomIntField.CUSTOM_INT_5,
+                Quadruple.of(customInt5In, customInt5NotIn, customInt5Within, customInt5NotWithin),
+                Quadruple.of(customInt5From, customInt5FromNot, customInt5To, customInt5ToNot)),
+            Triplet.of(
+                TaskCustomIntField.CUSTOM_INT_6,
+                Quadruple.of(customInt6In, customInt6NotIn, customInt6Within, customInt6NotWithin),
+                Quadruple.of(customInt6From, customInt6FromNot, customInt6To, customInt6ToNot)),
+            Triplet.of(
+                TaskCustomIntField.CUSTOM_INT_7,
+                Quadruple.of(customInt7In, customInt7NotIn, customInt7Within, customInt7NotWithin),
+                Quadruple.of(customInt7From, customInt7FromNot, customInt7To, customInt7ToNot)),
+            Triplet.of(
+                TaskCustomIntField.CUSTOM_INT_8,
+                Quadruple.of(customInt8In, customInt8NotIn, customInt8Within, customInt8NotWithin),
+                Quadruple.of(customInt8From, customInt8FromNot, customInt8To, customInt8ToNot)))
+        .forEach(
+            triplet -> {
+              TaskCustomIntField customField = triplet.getLeft();
+              Optional.ofNullable(triplet.getMiddle().getFirst())
+                  .ifPresent(wrap(l -> query.customIntAttributeIn(customField, l)));
+              Optional.ofNullable(triplet.getMiddle().getSecond())
+                  .ifPresent(wrap(l -> query.customIntAttributeNotIn(customField, l)));
+              Optional.ofNullable(triplet.getMiddle().getThird())
+                  .map(this::extractIntIntervals)
+                  .ifPresent(wrap(l -> query.customIntAttributeWithin(customField, l)));
+              Optional.ofNullable(triplet.getMiddle().getFourth())
+                  .map(this::extractIntIntervals)
+                  .ifPresent(wrap(l -> query.customIntAttributeNotWithin(customField, l)));
+              Integer from = triplet.getRight().getFirst();
+              Integer to = triplet.getRight().getThird();
+              if (from != null || to != null) {
+                query.customIntAttributeWithin(customField, new IntInterval(from, to));
+              }
+
+              Optional.ofNullable(triplet.getRight().getSecond())
+                  .ifPresentOrElse(
+                      wrap(
+                          l ->
+                              query.customIntAttributeNotWithin(
+                                  customField, new IntInterval(l, triplet.getRight().getFourth()))),
+                      () ->
+                          Optional.ofNullable(triplet.getRight().getFourth())
+                              .ifPresent(
+                                  wrap(
+                                      l ->
+                                          query.customIntAttributeNotWithin(
+                                              customField, new IntInterval(null, l)))));
+            });
```

Comment:
apply same logic like you did with "from" and "to"

Reply:
Ok :)